### DISCUSSION TITLE
Options Greek Calculations

### DISCUSSION THREAD

**Discussion by myalgomate** [DaveSkender/Stock.Indicators#887] [Ideas]
Hi Dave,

Greetings for the day.

Recently came across an essential thing for options which is Greek calculation.
It has a lot of importance and if we added it to our library it would be an exclusive piece of thing.

There are some reference for C# on git but is not updated and very limited things are available.
https://github.com/stochasticquant/black_scholes_option_pricing

If we add it to our library it would be the best thing.

**Comment by DaveSkender** [comment]
I’m open to exploring this a bit more.  The challenge with some of the Greeks is that there’s a time to expire factor, dividends, and risk free rates that complicate things … I’ve been down this road, trying to do arbitrage trades.

**Reply by myalgomate** [reply]
I can help you with all the needful formulas and calculations.
I have a pretty good sample project in C#.

**Reply by DaveSkender** [reply]
I'll look at your code, but in the end I'll just pull out my textbooks on these.  All the Greeks are well known mathematical formulas.
